### ISSUE TITLE
Tables

### ISSUE THREAD

**Issue by Patmando73** [Patmando73/-sinatra-Event-organizer#9]
### users

| id (primary key) | name (text) |
| --- | --- |
| 1 | "Jim" |
| 2 | "Patrick" |
| 3 | "Nick" |
| 4 | "Sam" |
### places

| id (primary key) | name (text) | attractions (text) | age_requirement (number) | hours_of_operation (text) | address (text) | parking_garage (boolean) |
| --- | --- | --- | --- | --- | --- | --- |
| 1 | PizzaHut | Pizza | 1 | 9:00 - 9:00 | 2425 N 84th St | true |
| 2 | Scorz | Bowling | 1 | 9:00 - 8:00 | 831 Main St. | false |
| 3 | Speilbound | Boardgames | 1 | 3:00 - 2:00 | Some cool St | true |
### likes

| id (primary key) | user_id (foriegn key) | place_id (foriegn key) | rating (number) |
| --- | --- | --- | --- |
| 1 | 3 | 1 | 4 |
| 2 | 1 | 2 | 5 |
| 3 | 3 | 1 | 3 |
| 4 | 4 | 3 | 5 |
### Overview.

The users table holds a primary key and the name of the user. The places table holds a primary key, name of the place, Attractions of that place such as happy hours, age requirements, hours of operation, the places address, and a boolean for if there is a parking garage. The final table is a Likes table which holds a primary key, a user id, a place id, and a rating the user gave to that place.

![sinatra 1](https://cloud.githubusercontent.com/assets/12161271/8337465/dcb7bb5c-1a70-11e5-944c-6ab1b97d3d9c.jpg)

**Comment by sumeetjain** [comment]
Needs to follow all rules for naming. https://github.com/omahacodeschool/wiki/blob/master/ruby/naming-things.md

**Comment by sumeetjain** [comment]
- [ ] Need to describe relations. (For example "A student has many questions.")

**Comment by sumeetjain** [comment]
Relationships are incorrect. Try drawing them out (like we do in class) and posting an image.

**Comment by sumeetjain** [comment]
Why does a user have many places? I get that they can 'like' many places, but why is there another relationship indicated between User and Place?

**Comment by Patmando73** [comment]
Users post the places for others to see and like/rate. So users can have many places they have posted

**Comment by sumeetjain** [comment]
In that case, you're missing a FK.

**Comment by Patmando73** [comment]
Yes! Sorry you and sam just pointed that out at the same time.. will fix that!
